DuckDuckGo/CenteredSearchHomeCell.swift

```diff
@@ -74,6 +75,7 @@ class CenteredSearchHomeCell: UICollectionViewCell {
     }
 
     @objc func onTap() {
+        Pixel.fire(pixel: .homeScreenSearchTapped)
```

Comment:
This fires for both 'centered search' and 'centered search w/ favorites' modes - is that intended (asking since this project is about making favorites the default, so I worry about possibility if misinterpreting the data afterwards)?

Reply:
I think that's ok as we'll see if people change the settings and then use the centered search.  Also I want to keep the separation of concerns, this thing shouldn't know about other renderers that are enabled.